x-pack/plugins/alerting/server/plugin.ts

```diff
@@ -462,6 +464,21 @@ export class AlertingPlugin {
     scheduleAlertingHealthCheck(this.logger, this.config, plugins.taskManager);
     scheduleApiKeyInvalidatorTask(this.telemetryLogger, this.config, plugins.taskManager);
 
+    plugins.actions.getUnsecuredActionsClient().then((unsecuredActionsClient) => {
```

Comment:
This is testing how it might be used from another plugin, with a preconfigured connector id of `gmail`.